README.md

```diff
@@ -908,6 +908,26 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
+     <td><code>bigNumericFieldsPrecision</code>
+     </td>
+     <td>A config to specify the precision to use while reading all the BigNumeric fields of a table.
```

Comment:
Should be `An alternative default precision for BigNumeric fields, as the BigQuery default is too wide for Spark. Values can be between 1 and 38.  This default is used only when the field has an unparameterized BigNumeric type.`

Reply:
Fixed